Creiwry/kitten_project
# fix(frontend): fix display error for edit cart item form

### What does this PR do?
It fixes display errors for the edit cart item form, by updating the route used for the form.
deletes and updates links on the edit view which were not working or redirecting to pages that don't exist

### Where should the reviewer start?
in the cart_items/edit and cart_items/_form views

### How should this be manually tested?
Try to access the edit cart item from the user profile in your browser. No errors should be encountered.
